### PR TITLE
indicators for all.

### PR DESCRIPTION
**Pull request by abenari**
added some missing progress indicators (spinners) to hosts and hosts group forms.

### CODE REVIEW

File: app/views/hosts/_form.html.erb [hunk 1]
```diff
@@ -40,23 +40,24 @@
...
-         :onchange => 'computeResourceSelected(this);'} if SETTINGS[:unattended] && @host.new_record? || @host.compute_resource_id %>
+         :onchange => 'computeResourceSelected(this);',
+         :help_inline => :indicator } if SETTINGS[:unattended] && @host.new_record? || @host.compute_resource_id %>
 
       <%= select_f f, :hostgroup_id, accessible_hostgroups, :id, :to_label,
         { :include_blank => true},
         { :onchange => 'hostgroup_changed(this);' , :'data-host-id' => @host.id,
-          :'data-url' => @host.new_record? ? process_hostgroup_hosts_path : hostgroup_or_environment_selected_hosts_path ,
-          :help_inline => image_tag('/assets/spinner.gif', :id => 'hostgroup_indicator', :class => 'hide').html_safe } %>
+          :'data-url' => (@host.new_record? || @host.type_changed?) ? process_hostgroup_hosts_path : hostgroup_or_environment_selected_hosts_path ,
```
Comment (domcleal) [inline app/views/hosts/_form.html.erb:55]: When's `@host.type_changed` used, for converting discovery hosts?

Comment (abenari) [inline app/views/hosts/_form.html.erb:55]: Yep.

**Comment by domcleal** [comment]
I'm happy with this, much cleaner.  Could you file a bug please and amend the commit msg?  Thanks.

**Comment by abenari** [comment]
added issue tracking #

File: app/views/hosts/_form.html.erb (hunk 1 above)
Comment (ohadlevy) [inline app/views/hosts/_form.html.erb:55]: ^^ @GregSutcliffe :+1:

**Comment by domcleal** [comment]
Thanks @abenari, merged as 89b9ecb07.

**Pull request closed without merging**